lib/APPRemoteControl/App.h

```diff
@@ -94,71 +95,95 @@ class App
      */
     void loop();
 
+    /**
+     * Handle remote commands received via SerialMuxProt.
+     *
+     * @param[in] cmd Command to handle.
+     */
+    void handleRemoteCommands(const Command& cmd);

```

Comment:
According to the comment, it handles several commands. But according to the parameters, it handles just one command at once.